tests/unit/browser/test_webelem.py

```diff
@@ -325,6 +327,38 @@ def test_debug_text(self, elem, xml, expected):
         elem._elem.toOuterXml.return_value = xml
         assert elem.debug_text() == expected
 
+    def test_remove_blank_target(self):
+        elem = get_webelem(tagname='a', attributes={'target': '_blank'})
+        elem.remove_blank_target()
+        assert elem._elem.attribute('target') == '_top'
+
+        for target in ['_self', '_parent', '_top', '']:
+            elem = get_webelem(tagname='a', attributes={'target': target})
+            elem.remove_blank_target()
+            assert elem._elem.attribute('target') == target
+
+        elem = get_webelem(tagname='a')
+        elem.remove_blank_target()
+        assert 'target' not in elem
+
+        elem = get_webelem(tagname='a', attributes={'target': '_blank'})
+        elem_child = get_webelem(tagname='img', parent=elem._elem)
+        elem_child._elem.encloseWith(elem._elem)
+        elem_child.remove_blank_target()
+        assert elem._elem.attribute('target') == '_top'
+
+        elem[0] = get_webelem(tagname='div')
+        for i in range(1, 5):
+            elem[i] = get_webelem(tagname='div', parent=elem[i-1])
+            elem[i]._elem.encloseWith(elem[i-1]._elem)
+        elem[4].remove_blank_target()
+        for i in range(5):
+            assert 'target' not in elem[i]
+
+        elem = get_webelem(tagname='div')
+        elem.remove_blank_target()
+        assert 'target' not in elem
```

Comment:
I mean that you could something like this following the example above:

```python
    @pytest.mark.parametrize('tagname', ['a', 'div'])
    def test_no_target(self, tagname):
        elem = get_webelem(tagname=tagname)
        elem.remove_blank_target()
        assert 'target' not in elem
```

to have both tests combined into one parametrized one